planner/core/preprocess.go

```diff
@@ -232,6 +239,10 @@ func (p *preprocessor) Leave(in ast.Node) (out ast.Node, ok bool) {
 				x.Args[0] = ast.NewValueExpr(0, "", "")
 			}
 		}
+
+		if x.FnName.L == ast.NextVal || x.FnName.L == ast.LastVal || x.FnName.L == ast.SetVal {
+			p.flag &= ^inSequenceFunction
+		}
```

Comment:
Test this by executing a SQL with both sequence and normal table.
e.g. `select seq.nextval(), t.id from t`.

Reply:
addressed